docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>I wonder if it would make more sense to have the long name first and then followed by the short parts</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Yes, I think that's a good idea too, changing the ordering</p>